.github/workflows/causal_lm_cpp.yml

```diff
@@ -432,6 +432,19 @@ jobs:
           assert predicted_greedy == predicted_speculative
           "
           echo "Alan Turing was a" passed
+      - name: run and compare python sample
+        run: |
+          source ./ov/setupvars.sh
+          && PYTHONPATH=./build/:$PYTHONPATH timeout 25s
```

Comment:
use `env` statement similar to https://github.com/openvinotoolkit/openvino.genai/blob/c8ac7cb540753d03ca0966d84e46899f23a90dbf/.github/workflows/stable_diffusion_1_5_cpp.yml#L95